### PR TITLE
Cocktails patch

### PR DESCRIPTION
**Pull request by Wraithcraft**
Adds Champagne, Bitters, Mint Syrup and 11 different cocktails. All of the spritework is awful. I'm sorry about that.

Cocktails: Champagne Mojito, The Classic, Cork Popper, French 75, Muscovite Mule, Orange Mimosa, Pink Gin, Pink Gin and Tonic, Pirate Punch, Planters Punch and Southside Royale.

### CODE REVIEW

**Comment by SoundScopes** [comment]
You'll want to rebase this into the development branch.
Also your changelog is making Travis fail. Make sure you list the change type on the same line as the log description.
`- rscadd:  Stuff
- rscadd:  More Stuff`

File: html/changelogs/wraithcraft-cocktails-patch.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+author: wraithcraft
+delete-after: True
+changes: 
+  - rscadd: 
+  - Added Champagne, Mint Syrup and Bitters. (Now avaliable in your local booze-o-mat)
```
Comment (Lohikar) [inline html/changelogs/wraithcraft-cocktails-patch.yml]: This changelog is invalid, should be:
```yml
- rscadd: "Added Champagne, Mint Syrup and Bitters. (Now avaliable in your local booze-o-mat)"
- rscadd: "Added Champagne to the booze dispenser."
- rscadd: "Added 11 new (moderately thought out) cocktails."
```

File: code/modules/reagents/reagent_containers/food/drinks/bottle.dm [hunk 2]
```diff
@@ -229,6 +229,33 @@
 		..()
 		reagents.add_reagent("nothing", 100)
 
+/obj/item/weapon/reagent_containers/food/drinks/bottle/bitters
+	name = " Angstra aromatic bitters"
```
Comment (Lohikar) [inline code/modules/reagents/reagent_containers/food/drinks/bottle.dm]: Extra space here.

File: code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm [hunk 3]
```diff
@@ -2175,6 +2289,57 @@
...
+	strength = 25
+	taste_description = "bitter christmas tree"
+
+	glass_icon_state = "pinkgin"
+	glass_name = "glass of pink gin"
+	glass_desc = "What an eccentric cocktail."
+	glass_center_of_mass = list("x"=16, "y"=9)
+
+/datum/reagent/ethanol/pinkgintonic
+	name = "Pink gin and Tonic."
```
Comment (skull132) [inline code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm]: Consistency in capitalization.

File: code/modules/reagents/reagent_containers/food/drinks/bottle.dm [hunk 4]
```diff
@@ -229,6 +229,33 @@
 		..()
 		reagents.add_reagent("nothing", 100)
 
+/obj/item/weapon/reagent_containers/food/drinks/bottle/bitters
+	name = "Angstra aromatic bitters"
```
Comment (skull132) [inline code/modules/reagents/reagent_containers/food/drinks/bottle.dm]: Capitalization.

File: code/modules/reagents/reagent_containers/food/drinks/bottle.dm [hunk 5]
```diff
@@ -229,6 +229,33 @@
...
+	name = "Tailfeather's Bubbliest champagne"
+	desc = "A rather fancy bottle of champagne, fit for collecting and storing in a cellar for decades."
+	icon_state = "champagnebottle"
+	center_of_mass = list("x"=16, "y"=4)
+	New()
+		..()
+		reagents.add_reagent("champagne",100)
+
+/obj/item/weapon/reagent_containers/food/drinks/bottle/mintsyrup
+	name = "Wintergreen mint syrup"
```
Comment (skull132) [inline code/modules/reagents/reagent_containers/food/drinks/bottle.dm]: Capitalization.

**Pull request merged**